### PR TITLE
Multi-site configuration for E-Learning

### PR DESCRIPTION
**Pull request by DavidLiedle**
This PR introduces configuration file changes setting up a multi-site configuration to allow for the hosting of learning.ezplatform.com as a discrete site, distinct from the root ezplatform.com. It presupposes fixed ID numbers that will need to be edited last-minute before merging.

### CODE REVIEW

**Comment by dspe** [comment]
@SylvainGuittard Just to be clear about the content:
1/ Should elearning part be create under to root location at the same depth than ezplatform.com
2/ Or should elearning be a part of ezplatform.com subtree 
Of course we could limit the access via a section if needed by example

File: app/config/ezplatform.yml [hunk 1]
```diff
@@ -37,6 +41,20 @@ ezpublish:
...
+            # Pool to use for cache, needs to be different per repository (database).
+            cache_pool_name: '%cache_pool%'
+            # These reflect the current installers, complete installation before you change them. For changing var_dir
+            # it is recommended to install clean, then change setting before you start adding binary content, otherwise you'll
+            # need to manually modify your database data to reflect this to avoid exceptions.
+            var_dir: var/learning
+            # HttpCache purge server(s) setting, eg Varnish, for when ezpublish.http_cache.purge_type is set to 'http'.
+            http_cache:
+                purge_servers: ["%purge_server%"]
+            design: main
```
Comment (andrerom) [inline app/config/ezplatform.yml]: should they use different designs?

eg:
com: [com, ezplatform]
learn: [learn, ezplatform]

**Comment by SylvainGuittard** [comment]
@dspe We discussed and selected a multi-site approach because the elearning platform will have several languages. And ezplatform.com not at all.
If you don't translate the content on the elearning platform, I am fine to put the content under ezplatform.com

File: app/config/ezplatform.yml (hunk 1 above)
Comment (SylvainGuittard) [reply app/config/ezplatform.yml]: You are totally right. We will bring the themes when we will submit the PR for the templates :)

**Pull request closed without merging**